src/etc/schema.graphqls

```diff
@@ -376,3 +383,30 @@ type AppliedTaxItemRate {
 extend input CartAddressInput {
     vat_id: String
 }
+
+type CustomizableFileOption implements CustomizableOptionInterface {
+    value: CustomizableFileValue
+}
+
+type CustomizableFieldValue {
+    file_extension: String @doc(description: "Option file extensions (If type file).")
+    price: Float
+    price_type: PriceTypeEnum
+    sku: String
+}
+
+type Stores {
+    stores: [Store]
+}
+
+type Store {
+    city: String
+    country: String
+    description: String
+    name: String
+    phone: String
+    pickup_location_code: String
+    postcode: String
+    region: String
+    street: String
```

Comment:
Why to redeclare every field here, is not this an extension of that type?

Reply:
We don't re-declare it. We declare it, because this is custom stuff (we don't extend already existing one)